src/index.md

```diff
@@ -36,50 +123,117 @@ PackageDefinition(
+    ]
+),
 PackageDefinition(
     "DocumenterMarkdown", "https://github.com/JuliaDocs/DocumenterMarkdown.jl",
     [
-        "README" => "https://github.com/JuliaDocs/DocumenterMarkdown.jl#documentermarkdown",
+        "stable" => "https://DocumenterMarkdown.juliadocs.org/stable/",
+        "dev" => "https://DocumenterMarkdown.juliadocs.org/dev/",
     ],
     [
         "https://codecov.io/gh/JuliaDocs/DocumenterMarkdown.jl/branch/master/graph/badge.svg" => "https://codecov.io/gh/JuliaDocs/DocumenterMarkdown.jl",
     ]
 ),
 PackageDefinition(
-    "DocumenterTools", "https://github.com/JuliaDocs/DocumenterTools.jl",
+    "DocumenterMermaid", "https://github.com/JuliaDocs/DocumenterMermaid.jl",
     [
-        "README" => "https://github.com/JuliaDocs/DocumenterTools.jl#documentertools",
+        "stable" => "https://juliadocs.org/DocumenterMermaid.jl/stable/",
+        "dev" => "https://juliadocs.org/DocumenterMermaid.jl/dev/",
     ],
     [
-        "https://codecov.io/gh/JuliaDocs/DocumenterTools.jl/branch/master/graph/badge.svg" => "https://codecov.io/gh/JuliaDocs/DocumenterTools.jl",
+        "https://codecov.io/gh/JuliaDocs/DocumenterMermaid.jl/branch/main/graph/badge.svg" => "https://codecov.io/gh/JuliaDocs/DocumenterMermaid.jl",
     ]
 ),
 PackageDefinition(
-    "DocStringExtensions", "https://github.com/JuliaDocs/DocStringExtensions.jl",
+    "DocumenterTools", "https://github.com/JuliaDocs/DocumenterTools.jl",
     [
-        "stable" => "https://DocStringExtensions.juliadocs.org/stable/",
-        "dev" => "https://DocStringExtensions.juliadocs.org/dev/",
+        "README" => "https://github.com/JuliaDocs/DocumenterTools.jl",
     ],
     [
-        "https://codecov.io/gh/JuliaDocs/DocStringExtensions.jl/branch/master/graph/badge.svg" => "https://codecov.io/gh/JuliaDocs/DocStringExtensions.jl",
+        "https://codecov.io/gh/JuliaDocs/DocumenterTools.jl/branch/master/graph/badge.svg" => "https://codecov.io/gh/JuliaDocs/DocumenterTools.jl",
     ]
 ),
 PackageDefinition(
     "Highlights", "https://github.com/JuliaDocs/Highlights.jl",
     [
-        "stable" => "https://Highlights.juliadocs.org/stable/",
-        "dev" => "https://Highlights.juliadocs.org/dev/",
+        "stable" => "https://highlights.juliadocs.org/stable/",
+        "dev" => "https://highlights.juliadocs.org/dev/",
     ],
     [
         "https://codecov.io/gh/JuliaDocs/Highlights.jl/branch/master/graph/badge.svg" => "https://codecov.io/gh/JuliaDocs/Highlights.jl",
     ]
 ),
+PackageDefinition(
+    "IOCapture", "https://github.com/JuliaDocs/IOCapture.jl",
+    [
+        "README" => "https://github.com/JuliaDocs/IOCapture.jl",
+    ],
+    [
+        "https://codecov.io/gh/JuliaDocs/IOCapture.jl/branch/master/graph/badge.svg" => "https://codecov.io/gh/JuliaDocs/IOCapture.jl",
```

Comment:
is outdated, someone needs to merge https://github.com/JuliaDocs/IOCapture.jl/pull/29 and then https://github.com/JuliaDocs/IOCapture.jl/pull/30